src/actions/sources/prettyPrint.js

```diff
@@ -124,6 +126,8 @@ export function togglePrettyPrint(sourceId: string) {
       })
     );
 
+    dispatch(closeTab(source));
```

Comment:
why do we need this?

Reply:
So, I picked it from https://github.com/devtools-html/debugger.html/issues/6947,

So close the original file when pretty printed one exists. It's a step at least.

Reply:
I don't think we want this, or if we do use it, do it in a separate PR so it's easy to back out if needed.